### PR TITLE
added page list widget

### PR DESCRIPTION
**Pull request by RenarsOzols**

### CODE REVIEW

**Comment by AndrejsFreiss** [comment]
Hello

File: app/code/Magebit/PageListWidget/Block/Widget/PageList.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<?php declare(strict_types=1);
+/**
+ * Copyright © Magebit, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
```
Comment (AndrejsFreiss) [inline app/code/Magebit/PageListWidget/Block/Widget/PageList.php]: Change the DocTypes like this, with strict_type declaration afterwards

```
<?php
/**
 * This file is part of the Magebit PageListWidget package.
 *
 * DISCLAIMER
 *
 * Do not edit or add to this file if you wish to upgrade Magebit_AmastyFaq
 * to newer versions in the future.
 *
 * @copyright Copyright (c) 2023 Magebit, Ltd. (https://magebit.com/)
 * @author    Magebit <info@magebit.com>
 * @license   MIT
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */

declare(strict_types=1); 
```

File: app/code/Magebit/PageListWidget/etc/module.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
```
Comment (AndrejsFreiss) [inline app/code/Magebit/PageListWidget/etc/module.xml:1]: Dock types in XML as well:
image

**Comment by RenarsOzols** [comment]
> Hello

Hello

File: app/code/Magebit/PageListWidget/registration.php [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?php
+
```
Comment (AndrejsFreiss) [inline app/code/Magebit/PageListWidget/registration.php:17]: Docktype here as well

File: app/code/Magebit/PageListWidget/view/frontend/templates/widget/page-list.phtml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<?php
```
Comment (AndrejsFreiss) [inline app/code/Magebit/PageListWidget/view/frontend/templates/widget/page-list.phtml:1]: Look into $escaper and what it does. 
All content must be escaped.
php echo can be shortened

image

**Review by AndrejsFreiss — CHANGES_REQUESTED** [review]

**Pull request merged**